code-of-conduct/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,92 @@
+
+**BOSC Organising Committee members and BOSC abstract reviewers**, even in cases when the OBF Code of Conduct doesn’t apply to the conference itself (depending on conference co-location it may or may not be adopted).
+
+**OBF Event fellowship awardees**, who are supported by OBF to attend an event and previous awardees when they represent OBF in an event or a community space.
+
+**Mentors, organisers, participants, project leaders, interns, and students**, when participating in CoFest, GSoC, GSoD, or similar OBF-umbrella activities for the duration of the event, internship, mentorship and similar occasions.
+
+**Communication and community engagement platforms**, such as project repositories on GitHub (issues, PRs or other contributions), mailing list, website (blogs and comment sections), Gitter channel and other communication platforms, workshops or training events, online forums such as Gitter chat rooms and Twitter.
+
+### 2.3 OBF Code of Conduct Committee
+
+The volunteer members designated and approved by the OBF Board to serve on the Code of Conduct Committee will be responsible for enforcing the Code of Conduct. Any voting members of the OBF community including the liaisons from the OBF Board can volunteer to sit in the Code of Conduct committee. They will be required to attend a training workshop on Code of Conduct report handling and enforcement as part of the onboarding process. We seek to build a committee including members outside of the OBF Board. Please contact the OBF board to express your interest to join this committee. The current members are listed in the OBF Code of Conduct document.
+
+Reports to the Code of Conduct Committee should be made by emailing `<dedicated Code of Conduct email>`. Currently appointed members to report any incident of violation of Code of Conduct are Bastian Greshake Tzovaras, Malvika Sharan and Yo Yehudi. They are also available to respond to any question and concerns regarding the Code of Conduct document and procedure.
+
+All reports will be reviewed by the Code of Conduct Committee and will be kept confidential.
+
+### 2.4 Expected Behavior
+
+All participants in our events and communications are expected to show respect and courtesy to others. All interactions should be professional regardless of platform: either online or in-person. In order to foster a positive and professional learning environment we encourage the following kinds of behaviors in all OBF events and platforms:
+
+* Use welcoming and inclusive language
+* Be respectful of different viewpoints and experiences
+* Gracefully accept constructive criticism
+* Focus on what is best for the community
+* Show courtesy and respect towards other community members
```

Comment:
This seems redundant with the Overview section. Does this really need to be in both places?

Reply:
No. I can edit that.